contrib/testing/.gitignore

```diff
@@ -0,0 +1,2 @@
+kind-custom.yaml
```

Comment:
nit: add explanatory comment

Reply:
thanks - good point. came up with some explanation that links it to the Make target.